manifests/plugin/remote_execution/cockpit.pp

```diff
@@ -0,0 +1,53 @@
+# Installs remote execution cockpit plugin
+class foreman::plugin::remote_execution::cockpit {
+  require foreman::plugin::remote_execution
+
+  $config_directory = '/etc/foreman/cockpit'
```

Comment:
Surprised there is not a variable that represents `/etc/foreman`

Reply:
So was I